doc/schemas/splice_init.request.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "channel_id",
+    "relative_amount"
+  ],
+  "properties": {
```

Comment:
OK, for new commands we want "added" fields here, not in the per-parameter descriptions.

One day we'll generate our usage docs from this, and then it will make perfect sense.

Reply:
Moved them up to the top level object in 10d85589893a03ca02fbb787ae717f93f775dcca